authors/apps/articles/models.py

```diff
@@ -60,3 +60,20 @@ def __str__(self):
 
     class Meta:
         ordering = ['-createdAt']
+
+
+class HighlightTextModel(models.Model):
+    user_id = models.ForeignKey(User, on_delete=models.CASCADE)
+    body = models.TextField()
+    article = models.ForeignKey(Article, related_name='highlight',
+                                on_delete=models.CASCADE, null=True)
+    selected_text = models.TextField(blank=True, null=True)
```

Comment:
For Textfield and Charfield, don't use null=True so that empty fields are saved with empty strings instead of NULL
https://docs.djangoproject.com/en/2.1/ref/models/fields/#django.db.models.Field.null

Reply:
okay